FlowCrypt/AppDelegate.swift

```diff
@@ -7,17 +7,30 @@ import AppAuth
 import UIKit
 import GTMAppAuth
 import FlowCryptCommon
+import Combine
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate, AppDelegateGoogleSesssionContainer {
     var blurViewController: BlurViewController?
     var googleAuthSession: OIDExternalUserAgentSession?
     let window = UIWindow(frame: UIScreen.main.bounds)
+    private var cancellable = Set<AnyCancellable>()
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         if application.isRunningTests {
             return true
         }
+        // Avoid race condition where the app might try to access keychain data before the device has decrypted it
```

Comment:
let's change this comment to:

`// When trying to initialize storage before protected data is available, existing storage may get corrupted due to a lost db encryption key. See https://github.com/FlowCrypt/flowcrypt-ios/issues/1373`